sdk/core/Azure.Core/tests/RetriableStreamTests.cs

```diff
@@ -371,16 +371,20 @@ private Task<int> ReadAsync(Stream stream, byte[] buffer, int offset, int length
         private static Stream SendTestRequest(HttpPipeline pipeline, long offset)
         {
             using Request request = CreateRequest(pipeline, offset);
-
```

Comment:
Need to validate what these are testing.

Reply:
This should be fine - it is relying on the fact that the mock framework allows overwriting the MockResponse.ContentStream and expects the transport not to replace the mock stream.